### PR TITLE
Create Discord version

### LINKED ISSUE #18: Make a discord version
Self-explanatory

### PR DESCRIPTION
**Pull request by diogoscf**
Closes #18 

Title says it

(Also fixed a few typos in README)

### CODE REVIEW

File: src/types_discord.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { CoreConfig } from "./types_core"
+
+/**
+ * Discord-specific configuration.
+ */
+export class DiscordConfig {
+  TOKEN: string
+
+  constructor() {
+    if (!process.env.TOKEN) throw new Error("TOKEN is missing!")
```
[the remark this answers is not in the document]
Comment (diogoscf) [reply src/types_discord.ts:10]: Yeah I was unsure whether or not to split them into two. Will do

[the remark this answers is not in the document]
Comment (diogoscf) [reply src/types_discord.ts]: Alright done

[the remark this answers is not in the document]
Comment (bartekpacia) [reply src/types_discord.ts]: change the message in `throw Error` also plz

**Comment by bartekpacia** [comment]
@diogoscf resolve conflicts and test again. I made some small changes to how `telegram` communicates with `core` and it will affect `discord` for sure.

File: src/types_discord.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (diogoscf) [reply src/types_discord.ts]: Done

**Comment by diogoscf** [comment]
Yeah I was hoping I could get this PR merged first so I wouldn't need to worry about conflicts 😢

**Comment by bartekpacia** [comment]
[F]

but it's only some package.json so I belive it won't be any kind of hard

**Comment by diogoscf** [comment]
Alright all done. Tested, and works

**Comment by bartekpacia** [comment]
nice, thank you

fixed #18

**Pull request merged**